APIs/schemas/edid.json

```diff
@@ -50,13 +42,21 @@
     },
     "display_type": {
       "description": "Digital display type in terms of supported subsampling modes",
-      "type": "string",
-      "enum": [
-        "RGB 4:4:4",
-        "RGB 4:4:4 + YUV 4:4:4",
-        "RGB 4:4:4 + YUV 4:2:2",
-        "RGB 4:4:4 + YUV 4:4:4 + YUV 4:2:2"
-      ]
+      "type": "array",
+      "minItems": 1,
+      "maxItems": 3,
+      "uniqueItems": true,
+      "contains": {
+        "const": "RGB444"
+      },
```

Comment:
This looked a little out of place, but I just want to confirm

> While the items schema must be valid for every item in the array, the contains schema only needs to validate against one or more items in the array.

ref: https://json-schema.org/understanding-json-schema/reference/array.html

➡️ Is your intention to allow an extra value to be present in the array?

Reply:
My intention was to restrict the array to always have the RGB value and the implementation satisfies the quote. When we moved to Receiver Caps values, I missed this const. It should be `RGB` and I will fix that. Thank you!

Reply:
I fixed its value in 3d9f28eb2630ae930e502ce6be6eccc863cc54dd.